### PR TITLE
infra deployment for rpc-proxy

### PR DESCRIPTION
**Pull request by optimisticben**
**Description**
Kubernetes resources to deploy an instance of the [rpc-proxy](https://github.com/ethereum-optimism/optimism/blob/develop/ops/docker/Dockerfile.rpc-proxy) (jsonrpc method filter)

And kustomizations for currently deployed instances in GCP.

Additional detail in the included README.

### CODE REVIEW

**Comment by tynes** [comment]
Would it be useful to be doing versioned github releases of the `ops` package?

File: ops/infra/rpc-proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+There are only 2 settings currently surfaced through the container.
+1. What methods to allow
+2. The address of the upstream sequencer
+
+Both are configured through environmental variables.
+```
+- name: SEQUENCER
+  value: sequencer:8545
+- name: ETH_CALLS_ALLOWED
```
Comment (snario) [inline ops/infra/rpc-proxy/README.md:13]: Couldn't a call also be `rollup_` prefixed? In which case `ETH_CALLS_ALLOWED` may be confusing. Also `eth_call` is a type of call, which may be confusing. Maybe a better name is `RPC_METHODS_ALLOWED`

Comment (optimisticben) [reply ops/infra/rpc-proxy/README.md]: Sure, updated!

**Comment by optimisticben** [comment]
Moving this to an infra repo, and maybe just getting rid of it.

**Pull request closed without merging**